scripts/qa_crowbarsetup.sh

```diff
@@ -3491,6 +3495,15 @@ function oncontroller_run_tempest()
     fi
     testr last --subunit | subunit-1to2 > tempest.subunit.log
 
+    if [ -n "$ostestroptions" ]; then
+        zypper -n in python-os-testr
+        ostestr $ostestroptions 2>&1 | tee ostestr.log
```

Comment:
@aplanas can we postpone that step for now? This is more of a workaround to be able to get some progress with magnum testing...